#! /bin/sh /usr/share/dpatch/dpatch-run
## 53_builtin-searchpath-fix.dpatch
##
## DP: adjust built-in search paths for kpathsea library to Debian standard

@DPATCH@
 texk/kpathsea/texmf.cnf |   14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

Index: texlive-bin-2009/texk/kpathsea/texmf.cnf
===================================================================
--- texlive-bin-2009.orig/texk/kpathsea/texmf.cnf	2009-10-08 23:03:54.000000000 +0000
+++ texlive-bin-2009/texk/kpathsea/texmf.cnf	2009-11-11 13:58:43.000000000 +0000
@@ -53,15 +53,15 @@
 
 % The tree containing the runtime files closely related to the specific
 % program version used:
-TEXMFMAIN = $SELFAUTOPARENT/texmf
+TEXMFMAIN = /usr/share/texmf
 
 % The main distribution tree:
-TEXMFDIST = $SELFAUTOPARENT/texmf-dist
+TEXMFDIST = /usr/share/texmf-texlive
 
 % A place for local additions to a "standard" texmf tree.
 % This tree is not used for local configuration maintained by
 % texconfig, it uses TEXMFCONFIG below.
-TEXMFLOCAL = $SELFAUTOPARENT/../texmf-local
+TEXMFLOCAL = /usr/local/share/texmf
 
 % TEXMFSYSVAR, where texconfig-sys stores variable runtime data.
 % With teTeX-3.0 or later, this must be set.
@@ -69,7 +69,7 @@
 %   TEXMFSYSVAR = $TEXMFMAIN
 % For using a separate tree:
 %   TEXMFSYSVAR = $SELFAUTOPARENT/texmf-var
-TEXMFSYSVAR = $SELFAUTOPARENT/texmf-var
+TEXMFSYSVAR = /var/lib/texmf
 
 % TEXMFSYSCONFIG, where texconfig-sys stores configuration data.
 % With teTeX-3.0 or later, this must be set.
@@ -77,7 +77,7 @@
 %   TEXMFSYSCONFIG = $TEXMFMAIN
 % For using a separate tree:
 %   TEXMFSYSCONFIG = $SELFAUTOPARENT/texmf-config
-TEXMFSYSCONFIG = $SELFAUTOPARENT/texmf-config
+TEXMFSYSCONFIG = /etc/texmf
 
 % User texmf trees are allowed as follows.
 % This used to be HOMETEXMF.
@@ -128,7 +128,7 @@
 % Where generated fonts may be written.  This tree is used when the sources
 % were found in a system tree and either that tree wasn't writable, or the
 % varfonts feature was enabled in MT_FEATURES in mktex.cnf.
-VARTEXFONTS = $TEXMFVAR/fonts
+VARTEXFONTS = /var/cache/fonts
 
 % Where to look for ls-R files.  There need not be an ls-R in the
 % directories in this path, but if there is one, Kpathsea will use it.
@@ -456,7 +456,7 @@
 % 
 % For security reasons, it is better not to include . in this path.
 %
-TEXMFCNF = {$SELFAUTOLOC,$SELFAUTODIR,$SELFAUTOPARENT}{,{/share,}/texmf{-local,}/web2c}
+TEXMFCNF = /usr/share/texmf/web2c;/usr/share/texmf-texlive/web2c;/usr/local/share/texmf/web2c
 
 # kpathsea 3.5.3 and later sets these at runtime. To avoid empty
 # expansions from binaries linked against an earlier version of the
